emcc.py

```diff
@@ -439,6 +439,10 @@ def get_cflags(user_args):
   if settings.WASM_WORKERS:
     cflags.append('-D__EMSCRIPTEN_WASM_WORKERS__=1')
 
+  if settings.ASYNCIFY:
+    # Note this macro doesn't care whether we're using JSPI or not.
+    cflags.append('-D__EMSCRIPTEN_HAS_ASYNCIFY__=1')
```

Comment:
Ah yes, the other reason this doesn't work today is that folks don't use `-sASYNCIFY` is the link-only setting.   So you will actually see a warning if you use it at compile time:

```
$ ./emcc -c ~/test/hello.c -sASYNCIFY
emcc: warning: linker setting ignored during compilation: 'ASYNCIFY' [-Wunused-command-line-argument]
```

Reply:
Ah, fair, I totally forgot about that. OK, `-DASYNCIFY` it is for now then. Thanks!